app/helpers/html_helper.php

```diff
@@ -5,3 +5,62 @@ function eh($string)
     if (!isset($string)) return;
     echo htmlspecialchars($string, ENT_QUOTES);
 }
+
+function readable_text($s) {
+    $s = htmlspecialchars($s, ENT_QUOTES);
+    $s = nl2br($s);
+    return $s;
+}
+
+// trim whitespace from $str and collapse adjacent whitespaces into one.
+function trim_collapse($str) {
+    $str = trim($str);
+    $str = preg_replace('/[[:space:]]{2,}/', ' ', $str);
+    return $str;
+}
+
+function redirect($str) {
+    header("Location: " . url($str));
+    die();
```

Comment:
ahh, you mean if redirection via header() does not work you'll use die() po?
by using header() here you're already loading the controller of the url you entered po